### PR TITLE
Switch to code generator sample files

### PR DESCRIPTION
**Pull request by JamesPaden**
These files have been programmatically generated and tested, except for the browser download example, which was manually generated and teested. Using them will work to standardize our code samples across all agents and documentation.

https://3.basecamp.com/3093825/buckets/1202046/todos/4410699987

### CODE REVIEW

File: examples/async.php [hunk 1]
```diff
@@ -6,57 +6,56 @@
...
 #
-# DocRaptor supports many options for output customization, the full list is
-# https://docraptor.com/documentation/api#api_general
+# DocRaptor supports many CSS and API options for output customization. Visit
+# https://docraptor.com/documentation/ for full details.
 #
 # You can run this example with: php async.php
-require __DIR__."/../vendor/autoload.php";
+
+require_once __DIR__ . "/vendor/autoload.php";
```
Comment (jason-o-matic) [inline examples/async.php:14]: I know you said the tests pass, but this path change seems like it would break running the tests in this repo?

**Review by jason-o-matic — COMMENTED** [review]
One open question.

File: examples/async.php (hunk 1 above)
Comment (JamesPaden) [reply examples/async.php:14]: This doesn't affect the [tests](https://github.com/DocRaptor/docraptor-php/tree/master/test) at all. But yes, it's not necessarily the correct path for all scenarios. It assumes the example script is at the root of your PHP directory; as if you just installed the agent and are now immediately trying to use the script.

Comment (JamesPaden) [reply examples/async.php:14]: To be more specific, I meant the tutorial-generator tests. I didn't run (or change in any way) the tests or code in this repo.

Comment (jason-o-matic) [reply examples/async.php:14]: Ah yeah, that makes sense.

**Review by jason-o-matic — APPROVED** [review]

**Pull request merged**